modules/tinymce/src/core/main/ts/api/util/ImageUploader.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
As mentioned yesterday, this new API needs some form of tests.

- [ ] Add some form of basic tests to ensure this public API works

Reply:
Done. Thanks for everyone help on that